solidity/tools/attestator.js

```diff
@@ -66,14 +65,17 @@ program
   .option('-e, --evm', 'sign with EVM signature format', false)
   .option('-s, --eos', 'sign with EOS signature format', true)
   .description('Gets an EOS event metadata')
-  .action((account, action, data, options) =>
+  .action((account, data, action, options) =>
```

Comment:
I'd keep the previous order if you don't mind, on evm version `data` is placed before the topics because they are a variable length argument so they have to stay at the end of the command line